src/components/calcite-modal/calcite-modal.tsx

```diff
@@ -35,6 +36,8 @@ export class CalciteModal {
   /** Optionally pass a function to run before close */
   @Prop() beforeClose: (el: HTMLElement) => Promise<void> = () =>
     Promise.resolve();
+  /** Display a close button within the Modal */
+  @Prop({ reflect: true }) closeButton = true;
```

Comment:
Since the default is to have the close button, maybe a prop like `disableCloseButton` might be better? Then it can be false by default. Otherwise, the `close-button` attribute would automatically be applied which is odd.

Reply:
@macandcheese Please advise.
I used the same property name as for popover for consistency, I thought consistency matter more.

Reply:
Yeah, the only difference is that the popover defaults to false so the attribute doesn't show up automatically.

Reply:
Yeah in this case lets go with `disable-*` since it also matches `disableEscape` prop - since the close button is there by default for modals users shouldn't have to add the attribute to see it IMO.

Reply:
OK I will update.